collectors/cgroups.plugin/sys_fs_cgroup.c

```diff
@@ -658,12 +672,17 @@ struct memory {
 // https://www.kernel.org/doc/Documentation/cgroup-v1/cpuacct.txt
 struct cpuacct_stat {
     int updated;
-    int enabled; // CONFIG_BOOLEAN_YES or CONFIG_BOOLEAN_AUTO
+    int enabled;            // CONFIG_BOOLEAN_YES or CONFIG_BOOLEAN_AUTO
+    int enabled_throttling; // CONFIG_BOOLEAN_YES or CONFIG_BOOLEAN_AUTO
 
     char *filename;
 
-    unsigned long long user;
-    unsigned long long system;
+    unsigned long long user;           // v1, v2(user_usec)
+    unsigned long long system;         // v1, v2(system_user)
+    unsigned long long nr_periods;     // v2 only
+    unsigned long long nr_throttled;   // v2 only
+    unsigned long long throttled_time; // v2 only (throttled_usec)
+    unsigned long long nr_throttled_perc;
```

Comment:
Following a discussion with Ilya... These variables are duplicated and it leads to unneeded `if`s for `rrddim_set`.